torchbenchmark/__init__.py

```diff
@@ -5,6 +5,7 @@
 import io
 import os
 import pathlib
+import platform
```

Comment:
Is this package being used in the code? Can we remove redundant dependency?

Reply:
Good catch - I have removed it.